clients/banking/src/components/CardWizard.tsx

```diff
@@ -415,30 +417,37 @@ export const CardWizard = ({
 
   const cardProducts = data?.projectInfo.cardProducts ?? [];
   const accountId = accountMembership.account?.id;
-  const [membersAfterCursor, setMembersAfterCursor] = useState<string | null>(null);
 
-  const [{ data: members, fetching }] = useQuery({
-    query: GetEligibleCardMembershipsDocument,
-    // not ideal but we need to keep the hook at top-level
-    variables: { accountId: accountId ?? "", first: 20, after: membersAfterCursor },
-    context: useMemo(() => ({ suspense: false }), []),
-  });
-
-  const hasMoreThanOneMember =
-    preselectedAccountMembership != null
-      ? false
-      : (members?.account?.allMemberships.totalCount ?? 0) > 1;
+  // not ideal but we need to keep the hook at top-level
+  const { data: members, setAfter: setMembersAfterCursor } = useUrqlPaginatedQuery(
+    {
+      query: GetEligibleCardMembershipsDocument,
+      variables: {
+        accountId: accountId ?? "",
+        first: 20,
+      },
+    },
+    [accountId],
+  );
 
   const canOrderPhysicalCard = step.cardFormat === "VirtualAndPhysical";
 
   if (accountId == null) {
     return <ErrorView />;
   }
 
-  if (members == null && fetching) {
+  if (members.isNotAsked() || members.isLoading()) {
     return <LoadingView color={colors.current[500]} />;
   }
 
+  const account = members
+    .get()
+    .map(({ account }) => account)
+    .getWithDefault(undefined);
+
+  const hasMoreThanOneMember =
```

Comment:
I would name this `canSelectMember` because we check we haven't preselected the membership

Reply:
It was already like that, I didn't changed it.